Dfe.Academies.External.Web.UnitTest/Pages/School/ApplicationJoinTrustReasonsModelTests.cs

```diff
@@ -0,0 +1,75 @@
+﻿using Dfe.Academies.External.Web.Pages.School;
+using Dfe.Academies.External.Web.Services;
+using Dfe.Academies.External.Web.UnitTest.Factories;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using Microsoft.AspNetCore.Mvc.Routing;
+using Microsoft.AspNetCore.Mvc.ViewFeatures;
+using Microsoft.Extensions.Logging;
+using Moq;
+using NUnit.Framework;
+using System.Threading.Tasks;
+
+namespace Dfe.Academies.External.Web.UnitTest.Pages.School;
+
+[Parallelizable(ParallelScope.All)]
+internal sealed class ApplicationJoinTrustReasonsModelTests
+{
+	/// <summary>
+	/// "draftConversionApplication" in temp storage
+	/// from previous step in the new application wizard
+	/// </summary>
+	/// <returns></returns>
+	[Test]
+	public async Task ApplicationJoinTrustReasonsModel___OnGetAsync___Valid()
```

Comment:
Test naming convention is:

MethodName___Scenario___ExpectedResult()

so this test could be:

```
OnGetAsync___Valid___NullErrors()
```